src/utils/hints/contracts.js

```diff
@@ -0,0 +1,6 @@
+export const contractsHints = {
+  contractId: 'A Smart Contract instance is an entity living on the block chain or in a state channel. Each instance has an address that can be used to call its entrypoints, either from another Smart Contract or in a ContractCallTx.',
```

Comment:
proposal:
- `Unique identifier for a Smart Contract instance living on the blockchain.`

I think the rest is not needed and on the explorer I would also remove the info about the contractId within a state channel because the contract instances living in a state channel are usually not visible and only known by the parties that created the channel.

Also, the rest of the hint is just too much info IMO.

Reply:
shouldn't we put "Smart Contract" lower case?
"block chain" --> "blockchain"

Reply:
interesting feedback there, but then lets go for it...